tests/integration/test_api/conftest.py

```diff
@@ -73,6 +75,20 @@ def configure_api_environment(get_configuration, request):
             control_service('restart')
 
 
+@pytest.fixture(scope='module')
+def clean_log_files(get_configuration, request):
+    """Reset the log files of the API and delete the rotated log files."""
+    def clean_log_files():
+        shutil.rmtree(os.path.join(WAZUH_PATH, 'logs', 'api'))
+        log_files = [API_LOG_FILE_PATH, API_JSON_LOG_FILE_PATH]
+        for log_file in log_files:
+            truncate_file(log_file)
+
```

Comment:
Clean logs at the start of the fixture too
```suggestion
clean_log_files()
```